app/src/main/java/com/example/sharewhatyoucanproject/login/OnBoardingFragment.kt

```diff
@@ -0,0 +1,47 @@
+package com.example.sharewhatyoucanproject.login
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.example.sharewhatyoucanproject.databinding.FragmentOnBoardingBinding
+import com.example.sharewhatyoucanproject.models.UserType
+import com.google.firebase.auth.FirebaseAuth
+
+class OnBoardingFragment : Fragment() {
+
+    private var _binding: FragmentOnBoardingBinding? = null
+    private val binding get() = _binding!!
```

Comment:
make sure to null out the binding in `onDestroyView`

Reply:
done